File: Java/Events/FullTicketInfo.java
package LTD_API;

public class FullTicketInfo
{
    private int ticketId;
    private String areaName;
    private String ticketName1;
    private String ticketName2;
    private decimal sellingPrice;
    private decimal faceValue;
    private bool restrictedView;
    private String description;
  
    public int getTicketId()
    {
        return ticketId;
    }
    public void setTicketId(int ticketId)
    {
        this.ticketId = ticketId;
    }

    public String getAreaName()
    {
        return areaName;
    }
    public void setAreaName(String areaName)
    {
        this.areaName = areaName;
    }

    public String getTicketName1()
    {
        return ticketName1;
    }
    public void setTicketName1(String ticketName1)
    {
        this.ticketName1 = ticketName1;
    }

    public String getTicketName2()
    {
        return ticketName2;
    }
    public void setTicketName2(String ticketName2)
    {
        this.ticketName2 = ticketName2;
    }

    public decimal getSellingPrice()
    {
        return sellingPrice;
    }
    public void setSellingPrice(decimal sellingPrice)
    {
        this.sellingPrice = sellingPrice;
    }

    public decimal getFaceValue()
    {
        return faceValue;
    }
    public void setFaceValue(decimal faceValue)
    {
        this.faceValue = faceValue;
    }

    public bool getRestrictedView()
    {
        return restrictedView;
    }
    public void setRestrictedView(bool restrictedView)
    {
        this.restrictedView = restrictedView;
    }

    public String getDescription()
    {
        return description;
    }
    public void setDescription(String description)
    {
        this.description = description;
    }	
}

File: PHP/Events/FullTicketInfo.php
<?php
namespace LTD_API;

class FullTicketInfo
{
    private $ticketId;
    private $areaName;
    private $ticketName1;
    private $ticketName2;
    private $sellingPrice;
    private $faceValue;
    private $restrictedView;
    private $description;

    public function getTicketId()
    {
        return $this->ticketId;
    }
    public function setTicketId($ticketId)
    {
        $this->ticketId = $ticketId;
    }

    public function getAreaName()
    {
        return $this->areaName;
    }
    public function setAreaName($areaName)
    {
        $this->areaName = $areaName;
    }

    public function getTicketName1()
    {
        return $this->ticketName1;
    }
    public function setTicketName1($ticketName1)
    {
        $this->ticketName1 = $ticketName1;
    }

    public function getTicketName2()
    {
        return $this->ticketName2;
    }
    public function setTicketName2($ticketName2)
    {
        $this->ticketName2 = $ticketName2;
    }

    public function getSellingPrice()
    {
        return $this->sellingPrice;
    }
    public function setSellingPrice($sellingPrice)
    {
        $this->sellingPrice = $sellingPrice;
    }

    public function getFaceValue()
    {
        return $this->faceValue;
    }
    public function setFaceValue($faceValue)
    {
        $this->faceValue = $faceValue;
    }

    public function getRestrictedView()
    {
        return $this->restrictedView;
    }
    public function setRestrictedView($restrictedView)
    {
        $this->restrictedView = $restrictedView;
    }

    public function getDescription()
    {
        return $this->description;
    }
    public function setDescription($description)
    {
        $this->description = $description;
    }
}
